--- lshw-B.02.11/src/Makefile.orig	2007-07-20 00:42:20.000000000 +0200
+++ lshw-B.02.11/src/Makefile	2007-08-03 21:36:53.908682481 +0200
@@ -7,7 +7,7 @@
 SBINDIR=$(PREFIX)/sbin
 MANDIR=$(PREFIX)/share/man
 DATADIR=$(PREFIX)/share
-INSTALL?=install -p -s
+INSTALL?=install -p
 STRIP?=strip
 export DESTDIR
 export PREFIX
@@ -37,7 +37,9 @@
 core:
 	+make -C core all
 
-$(PACKAGENAME): core $(PACKAGENAME).o
+core/lib$(PACKAGENAME).a: core
+
+$(PACKAGENAME): core/lib$(PACKAGENAME).a $(PACKAGENAME).o
 	$(CXX) $(LDFLAGS) -o $@ $(PACKAGENAME).o $(LIBS)
 
 .PHONY: gui
@@ -47,7 +49,7 @@
 .PHONY: static
 static: $(PACKAGENAME)-static
 
-$(PACKAGENAME)-static: core core/lib$(PACKAGENAME).a $(PACKAGENAME).o
+$(PACKAGENAME)-static: core/lib$(PACKAGENAME).a $(PACKAGENAME).o
 	$(CXX) $(LDSTATIC) $(LDFLAGS) -o $@ $(PACKAGENAME).o $(LIBS)
 	$(STRIP) $@
 
