diff -ruN lshw-B.02.17.orig/src/core/fat.cc lshw-B.02.17/src/core/fat.cc
--- lshw-B.02.17.orig/src/core/fat.cc	2012-02-22 14:39:44.000000000 +0100
+++ lshw-B.02.17/src/core/fat.cc	2015-01-12 01:51:40.710213032 +0100
@@ -65,6 +65,7 @@
 			uint8_t magic[8];
 			uint8_t dummy2[192];
 			uint8_t pmagic[2];
+			uint8_t dummy3[256];
 		} __attribute__((__packed__)) fat;
 		struct fat32_super_block {
 			uint32_t fat32_length;
@@ -80,6 +81,7 @@
 			uint8_t magic[8];
 			uint8_t dummy2[164];
 			uint8_t pmagic[2];
+			uint8_t dummy3[256];
 		} __attribute__((__packed__)) fat32;
 		char sector[512];	// to make sure the whole struct is at least 512 bytes long
 	} __attribute__((__packed__)) type;
