diff -ruNp lshw-B.02.18.orig/src/gui/Makefile lshw-B.02.18/src/gui/Makefile
--- lshw-B.02.18.orig/src/gui/Makefile	2016-04-22 21:49:30.000000000 +0200
+++ lshw-B.02.18/src/gui/Makefile	2017-06-17 21:37:34.254783020 +0200
@@ -1,5 +1,7 @@
 PACKAGENAME?=lshw
 
+SQLITE?=0
+
 CXX?=$(CROSS_COMPILE)c++
 CC?=$(CROSS_COMPILE)cc
 STRIP?=strip
@@ -8,7 +10,7 @@
 DEFINES=-DPREFIX=\"$(PREFIX)\" -DSBINDIR=\"$(SBINDIR)\" -DMANDIR=\"$(MANDIR)\" -DDATADIR=\"$(DATADIR)\"
 GTKINCLUDES=$(shell pkg-config gtk+-2.0 --cflags)
 INCLUDES=-I../core $(GTKINCLUDES)
-CXXFLAGS=-g -Wall $(INCLUDES) $(DEFINES) $(RPM_OPT_FLAGS)
+CXXFLAGS += -g -Wall $(INCLUDES) $(DEFINES) $(RPM_OPT_FLAGS)
 ifeq ($(SQLITE), 1)
 	CXXFLAGS+= -DSQLITE $(shell pkg-config --cflags sqlite3)
 endif
@@ -18,10 +20,6 @@
 ifeq ($(SQLITE), 1)
 	LIBS+= $(shell pkg-config --libs sqlite3)
 endif
-LDFLAGS=
-ifneq ($(shell $(LD) --help 2| grep -- --as-needed), )
-	LDFLAGS+= -Wl,--as-needed
-endif
 
 OBJS = gtk-lshw.o callbacks.o engine.o print-gui.o stock.o
 SRCS = $(OBJS:.o=.c)
@@ -39,8 +42,7 @@ gtk-$(PACKAGENAME): $(OBJS) ../core/libl
 	$(CXX) $(LDFLAGS) -o $@ $^ $(LIBS)
 
 install: all
-	$(STRIP) gtk-$(PACKAGENAME)
-	
+
 clean:
 	rm -f $(OBJS) gtk-$(PACKAGENAME) gtk-lshw.glade.bak gtk-lshw.gladep.bak callbacks.c.bak callbacks.h.bak Makefile.bak
 
diff -ruNp lshw-B.02.18.orig/src/Makefile lshw-B.02.18/src/Makefile
--- lshw-B.02.18.orig/src/Makefile	2016-04-22 21:49:30.000000000 +0200
+++ lshw-B.02.18/src/Makefile	2017-06-17 21:34:48.390265690 +0200
@@ -23,14 +23,14 @@
 CXX?=$(CROSS_COMPILE)c++
 INCLUDES=-I./core/
 DEFINES=-DPREFIX=\"$(PREFIX)\" -DSBINDIR=\"$(SBINDIR)\" -DMANDIR=\"$(MANDIR)\" -DDATADIR=\"$(DATADIR)\" -DVERSION=\"$(VERSION)\"
-CXXFLAGS=-g -Wall -g $(INCLUDES) $(DEFINES) $(RPM_OPT_FLAGS)
+CXXFLAGS += -g -Wall -g $(INCLUDES) $(DEFINES) $(RPM_OPT_FLAGS)
 ifeq ($(SQLITE), 1)
 	CXXFLAGS+= -DSQLITE $(shell pkg-config --cflags sqlite3)
 endif
 ifeq ($(ZLIB), 1)
 	CXXFLAGS+= -DZLIB $(shell pkg-config --cflags zlib)
 endif
-LDFLAGS+=-L./core/ -g
+LDFLAGS+=-L./core/
 ifneq ($(shell $(LD) --help 2| grep -- --as-needed), )
 	LDFLAGS+= -Wl,--as-needed
 endif
@@ -47,13 +47,7 @@
 export LIBS
 export LDFLAGS
 
-ifeq ($(ZLIB), 1)
-DATAFILES = pci.ids.gz usb.ids.gz oui.txt.gz manuf.txt.gz pnp.ids.gz pnpid.txt.gz
-else
-DATAFILES = pci.ids usb.ids oui.txt manuf.txt pnp.ids pnpid.txt
-endif
-
-all: $(PACKAGENAME) $(PACKAGENAME).1 $(DATAFILES)
+all: $(PACKAGENAME) $(PACKAGENAME).1
 
 .cc.o:
 	$(CXX) $(CXXFLAGS) -c $< -o $@
@@ -63,18 +57,18 @@
 
 .PHONY: core
 core:
-	+make -C core all
+	$(MAKE) -C core all
 
 $(PACKAGENAME): core $(PACKAGENAME).o
 	$(CXX) $(LDFLAGS) -o $@ $(PACKAGENAME).o $(LIBS)
 
 .PHONY: po
 po:
-	+make -C po all
+	$(MAKE) -C po all
 
 .PHONY: gui
 gui: core
-	+make -C gui all
+	$(MAKE) -C gui all
 
 .PHONY: nologo
 nologo:
@@ -85,7 +79,6 @@
 
 $(PACKAGENAME)-static: core core/lib$(PACKAGENAME).a $(PACKAGENAME).o
 	$(CXX) $(LDSTATIC) $(LDFLAGS) -o $@ $(PACKAGENAME).o $(LIBS)
-	$(STRIP) $@
 
 .PHONY: compressed
 compressed: $(PACKAGENAME)-compressed
@@ -114,14 +107,13 @@
 pnpid.txt:
 	wget http://www-pc.uni-regensburg.de/hardware/TECHNIK/PCI_PNP/pnpid.txt
 
-install: all
+install:
 	$(INSTALL) -d -m 0755 $(DESTDIR)/$(SBINDIR)
 	$(INSTALL) -m 0755 $(PACKAGENAME) $(DESTDIR)/$(SBINDIR)
 	$(INSTALL) -d -m 0755 $(DESTDIR)/$(MANDIR)/man1
 	$(INSTALL) -m 0644 $(PACKAGENAME).1 $(DESTDIR)/$(MANDIR)/man1
 	$(INSTALL) -d -m 0755 $(DESTDIR)/$(DATADIR)/$(PACKAGENAME)
-	$(INSTALL) -m 0644 $(DATAFILES) $(DESTDIR)/$(DATADIR)/$(PACKAGENAME)
-	make -C po install
+	$(MAKE) -C po install
 
 install-gui: gui
 	$(INSTALL) -d -m 0755 $(DESTDIR)/$(SBINDIR)
@@ -134,8 +126,8 @@
 clean:
 	rm -f $(PACKAGENAME).o $(PACKAGENAME) $(PACKAGENAME)-static $(PACKAGENAME)-compressed
 	rm -f $(addsuffix .gz,$(DATAFILES))
-	make -C core clean
-	make -C gui clean
+	$(MAKE) -C core clean
+	$(MAKE) -C gui clean
 
 depend:
 	@makedepend -Y $(SRCS) 2> /dev/null > /dev/null
